### PR TITLE
Update the "future" episode

### PR DESCRIPTION
**Pull request by bast**
Based on discussion with @rkdarst

### CODE REVIEW

**Comment by bast** [comment]
I kept the certification part although a bit less relevant in the present context.

File: content/future.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- Start a study group
+
+---
+
+## Biggest open problems
+
+- How to give helpers and contributors more credit and visibility
+- How to promote/engage new members
+- We don't know how we should be in relation with Carpentries
+- What are we? Non-profit? Organization? Selling services?
```
Comment (rkdarst) [inline content/future.md:51]:
```suggestion
- What are we? Non-profit? Institution collaboration network? Selling services?
```
To me "organization" sounds similar to non-profit.  I added another option, network of institutions.

File: content/future.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+---
+
+## How to get involved in CodeRefinery
+
+There are many ways to get involved: <https://coderefinery.org/get-involved/>
+
+- Become a workshop helper
+- Become a CodeRefinery workshop instructor
+- Become a Carpentries workshop instructor
+- Contribute to teaching material: <https://github.com/coderefinery>
```
Comment (rkdarst) [inline content/future.md:37]:
```suggestion
- Contribute to teaching material: <https://github.com/coderefinery/>
```

File: content/future.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+- Funding and sustainability (but we have ideas: collaboration network)
+- Scalability and marketing question: How to reach more people?
+  ([Research Software Hour](https://researchsoftwarehour.github.io/), Twitch, streaming-only
+  workshops, recording, post-processing videos)
+
+---
+
+## Certification for CodeRefinery instructors
+
+We will set up a more formal process soon but for now you would need to:
```
Comment (rkdarst) [inline content/future.md:61]: Perhaps link https://coderefinery.github.io/manuals/instructor-intro/#from-helper-to-more

**Review by rkdarst — APPROVED** [review]
Some suggestions but looks good, with or without them.

**Comment by rkdarst** [comment]
Open question for me: does our collaboration network die once we go back to in-person?

File: content/future.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
+# Future
+
+## Activities
+
+### What is CodeRefinery?
+- Teaching tools and best practices for version control and reproducibility in modern research software development
+- Over 30 workshops to almost 1000 participants
+- Typical workshop: <https://coderefinery.github.io/2020-10-20-online/#schedule>
+- Lessons: <https://coderefinery.org/lessons/>
+- Carpentries membership
```
[the remark this answers is not in the document]
Comment (bast) [reply content/future.md:10]: And there is an "about" session. I agree it should be split but I am out of time to change the "about" session and it is too long to go over tomorrow. Next time.

**Pull request merged**